hypertrace-gradle-publish-maven-central-plugin/src/main/java/org/hypertrace/gradle/publishing/PublishMavenCentralPlugin.java

```diff
@@ -37,25 +39,31 @@ public class PublishMavenCentralPlugin implements Plugin<Project> {
   @Override
   public void apply(@Nonnull Project target) {
     project = target;
-    this.extension =
-      project.getExtensions().create(EXTENSION_NAME, HypertracePublishMavenCentralExtension.class);
-    this.applyMavenPublish();
-    this.applySigning();
-    this.applyWithJavadocJar();
-    this.applyWithSourcesJar();
-    this.addPublishRepository();
-    this.addPublications();
-    this.applyNexusStaging();
+    // Root projects only should configure nexus staging
+    if (project.equals(project.getRootProject())) {
```

Comment:
this is the main change. Before, it auto applied the nexus staging to root from any child that needed it. Gradle no longer likes one project to modify a parent, so now we support applying this explicitly to a parent.